index.html

```diff
@@ -569,26 +569,35 @@ <h3>Use Cases and Requirements</h3>
         </p>
 
         <p>
-A <dfn class="lint-ignore">conforming processor</dfn> is any algorithm realized
-as software and/or hardware that generates or consumes a <a>conforming
-document</a>. Conforming processors MUST produce errors when non-conforming
-documents are consumed.
+A <dfn class="lint-ignore">conforming issuer implementation</dfn> produces
+<a>conforming documents</a>, MUST include all required properties in the
+<a>conforming documents</a> that it produces, and MUST secure the <a>conforming
+documents</a> it produces using a securing mechanism as described in Section
+<a href="#securing-verifiable-credentials"></a>.
+        </p>
+
+        <p>
+A <dfn class="lint-ignore">conforming verifier implementation</dfn>
+consumes <a>conforming documents</a>, MUST perform <a>verification</a> on a
+<a>conforming document</a> as described in Section
+<a href="#securing-verifiable-credentials"></a>, MUST check that each
+required property satisfies the normative requirements for that property, and
+MUST produce errors when non-<a>conforming documents</a> are detected.
         </p>
 
         <p>
-This specification includes both required and optional fields. An <a>issuer</a>
-MUST include all required fields. A <a>verifier</a> SHOULD check that each
-required field satisfies the normative requirements for that field.
-Optional fields MAY
-be ignored by <a>issuers</a> and/or <a>verifiers</a>.
+This specification includes both required and optional properties. Optional
+properties MAY be ignored by <a>conforming issuer implementations</a> and/or
+<a>conforming verifier implementations</a>.
         </p>
 
         <p>
-<a>Verifiable credential</a> and <a>verifiable presentation</a> MUST be
-protected using a digital proof mechanism such as a digital signature. Having
-and verifying proofs, which may be dependent on the syntax of the proof (for
-example, using the JSON Web Signature of a JSON Web Token for proofing a key
-holder), are an essential part of processing <a>verifiable credentials</a> and
+<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be
+protected using a securing mechanism as described in Section
```

Comment:
Why was "digital proof mechanism" replaced with "securing mechanism"? This question does not affect my approval of this pull request; I am merely curious to know whether this is a matter of preference, or whether it is part of a change in terminology agreed upon by the VCWG.

Reply:
The "digital proof mechanism" was from the v1.0 and v1.1 days, and that's what the VCWG (at that time) had settled upon for the language. The VCWG v2.0 charter shifted to using the term "Securing": https://www.w3.org/2022/06/verifiable-credentials-wg-charter.html#deliverables

... and all of our "securing" specifications have "securing", either in the title or the subtitle. I still need to make an editorial pass over the entire spec to update the language, but it seems the new direction is "securing" and "securing mechanisms".